src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeActions/CodeActionRequestPriority.cs

```diff
@@ -11,24 +11,39 @@ internal enum CodeActionRequestPriority
     {
         /// <summary>
         /// No priority specified, all refactoring, code fixes, and analyzers should be run.  This is equivalent
-        /// to <see cref="Normal"/> and <see cref="High"/> combined.
+        /// to <see cref="Lowest"/>, <see cref="Low"/>, <see cref="Normal"/> and <see cref="High"/> combined.
         /// </summary>
         None = 0,
 
+        /// <summary>
+        /// Only lowest priority suppression and configuration fix providers should be run.  Specifically,
+        /// <see cref="T:IConfigurationFixProvider"/> providers will be run.
+        /// </summary>
```

Comment:
can you put a comment that says this is reserved and should never be used by any other fixers/refactorings?